src/agoradatatools/etl/transform.py

```diff
@@ -116,19 +117,15 @@ def calculate_distribution(df: pd.DataFrame, col: str, is_scored, upper_bound) -
     column with both 0 and that maximum value added to it.  We use the copy to calculate 
     distributions and bins, and subtract the values at the end
     """
-    distribution = df[col].append(pd.Series([0, upper_bound]), ignore_index=True)
+    distribution = pd.concat([df[col], pd.Series([0, upper_bound])], ignore_index=True)
```

Comment:
Nit: This is great - I know there are two other places that use `.append` in `create_proteomics_distribution_data`.  Should we switch those to`pd.concat` in this PR, or create a Jira ticket for that?

Reply:
Whichever @jaclynbeck-sage prefers works for me.

Reply:
The `create_proteomics_distribution_data` is using `.append` on a list, not a pandas dataframe, so these appends are ok to leave in!

Reply:
Thanks for clarifying!  Should have looked closer!